src/pointer.rs

```diff
@@ -1184,6 +1188,56 @@ const fn validate(value: &str) -> Result<&str, ParseError> {
     Ok(value)
 }
 
+/*
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+╔══════════════════════════════════════════════════════════════════════════════╗
+║                                                                              ║
+║                              TopicalParseError                               ║
+║                             ¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯                              ║
+╚══════════════════════════════════════════════════════════════════════════════╝
+░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░
+*/
+#[derive(Debug, PartialEq)]
+pub struct ParseBufError {
+    pub value: String,
+    pub source: ParseError,
```

Comment:
I don't think these need to/should be public

Reply:
Sounds good - I seem to have a tendency of over exposing fields.